### PR TITLE
add test manager binary

### PR DESCRIPTION
**Pull request by hopkiw**
depends on #100 (which depends on #98) 
add test manager and first test package

### CODE REVIEW

**Comment by hopkiw** [comment]
depends also on #98

File: imagetest/cmd/manager/main.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+func (l *logWriter) Write(b []byte) (int, error) {
+	l.log.Print(string(b))
+	return len(b), nil
+}
+
+func main() {
+	flag.Parse()
+	if *project == "" || *zone == "" || *images == "" {
+		log.Fatal("Must provide project, zone and images arguments")
+		return
```
Comment (ericdand) [inline imagetest/cmd/manager/main.go:37]: log.Fatal already bails out; you can drop this return. :)

File: imagetest/test_suites/image_validation/image_validation_test.go [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+		t.Errorf("hostname does not match metadata. Expected: %q got: %q", shortname, hostname)
+	}
+
+	// If hostname is FQDN then lots of tools (e.g. ssh-keygen) have issues
+	if strings.Contains(hostname, ".") {
+		t.Errorf("hostname contains '.'")
+	}
+}
+
+// TestCustomHostname tests the 'fully qualified domain name', using the logic in the `hostname` utility.
```
Comment (ericdand) [inline imagetest/test_suites/image_validation/image_validation_test.go:54]: Looks like you renamed this function and forgot to update the doc comment to match.

File: imagetest/test_suites/image_validation/image_validation_test.go [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+	}
+	hostname := strings.TrimRight(string(out), " \n")
+
+	if hostname != metadataHostname {
+		t.Errorf("hostname does not match metadata. Expected: %q got: %q", metadataHostname, hostname)
+	}
+}
+
+func TestArePackagesLegalToUse(t *testing.T) {
+	// This will be a bit longer to translate ;)
```
Comment (ericdand) [inline imagetest/test_suites/image_validation/image_validation_test.go:75]: Maybe leave a TODO here to mark it as intended to be implemented later?

**Review by ericdand — COMMENTED** [review]
/lgtm

LGTM, just a couple small things to clean up.

**Pull request merged**